tests/access/test_AccessControl.py

```diff
@@ -0,0 +1,178 @@
+import pytest
+from pathlib import Path
+from starkware.starknet.testing.starknet import Starknet
+from utils import (
+    Signer, TRUE, FALSE,
+    assert_event_emitted, assert_revert,
+    get_contract_def, cached_contract
+)
+
+DEFAULT_ADMIN_ROLE = 0
+SOME_OTHER_ROLE = 42
+
+signer = Signer(123456789987654321)
```

Comment:
```suggestion

signer = TestSigner(123456789987654321)
```
I believe this is why it's failing. Just import TestSigner and tests should be good to go